### PR TITLE
e2e

### PR DESCRIPTION
**Pull request by fluidvanadium**
added readme including commandline android instructions
also some testIDs

### CODE REVIEW

**Comment by zancas** [comment]
Hi @juanky201271 can you review this PR?

File: e2e/03_new_wallet.test.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const { log, device, by, element } = require('detox');
```
Comment (zancas) [inline e2e/03_new_wallet.test.js:1]: What's with the numerical prefixes on the file names?

**Review by zancas — DISMISSED** [review]
Version numbers in filenames smell funny to me, is there work here, that `git` needs to take care of?

**Comment by fluidvanadium** [comment]
> Version numbers in filenames smell funny to me, is there work here, that `git` needs to take care of?

they're not version numbers. they are intended to get jest to run the tests in a specific order. right now that's not WAT

File: e2e/03_new_wallet.test.js (hunk 1 above)
Comment (juanky201271) [reply e2e/03_new_wallet.test.js:1]: I don't really understand what you mean... I don't see any numerical in `const { log, device, by, element } = require('detox');`... sorry in advance.

**Review by juanky201271 — DISMISSED** [review]
I this is great, I don't see any problem...

File: e2e/03_new_wallet.test.js (hunk 1 above)
Comment (juanky201271) [reply e2e/03_new_wallet.test.js:1]: I'm going to resolve this in order to merge it.

**Comment by juanky201271** [comment]
@zancas I don't think there are any change to do, IMO... this is ready to merge.

**Comment by juanky201271** [comment]
@fluidvanadium sorry I did everything to merge this....

**Review by juanky201271 — DISMISSED** [review]
This is even better than the last time I approved it... It's like wine, get better with time...

**Review by juanky201271 — APPROVED** [review]
great!

**Pull request merged**